SampleApplications/Samples/NetCoreConsoleServer/Opc.Ua.SampleServer.Config.xml

```diff
@@ -177,6 +178,19 @@
     <MaxTrustListSize>0</MaxTrustListSize>
     <MultiCastDnsEnabled>false</MultiCastDnsEnabled>
 
+    <ReverseConnect>
+      <Clients>
+        <!--
+        <ReverseConnectClient>
+          <EndpointUrl>opc.tcp://localhost:65300</EndpointUrl>
+          <Timeout>30000</Timeout>
+        </ReverseConnectClient>
+        -->
```

Comment:
[Question] Why is only the Reverse Connect Client commented out?

Reply:
I think it shows better how to add clients...and for programmatic adding endpoints they still got to use the defaults from here

---
In reply to: [432388963](https://github.com/OPCFoundation/UA-.NETStandard/pull/998#discussion_r432388963) [](ancestors = 432388963)